.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.